File: dataset_reduzido/codenet/Python/Code/s702834515.py
"""
i が 3 でも 5 でも割り切れるなら、ai=FizzBuzz
そうではなく i
が 3 で割り切れるなら、ai=Fizz
そうではなく i
が 5 で割り切れるなら、ai=Buzz
そうではないなら、ai=i
"""
n=int(input())
r=0
for ii in range(n):
  if (ii+1)%3!=0 and (ii+1)%5!=0:
    r+=(ii+1)
print(r)

File: output/Llama-2-100/codenet/Python/Java/s702834515.java
public class s702834515 {
  public static void main(String[] args) {
    int n = Integer.parseInt(args[0]);
    int r = 0;
    for (int ii = 0; ii < n; ii++) {
      if ((ii + 1) % 3 == 0 && (ii + 1) % 5 == 0) {
        r += (ii + 1);
      } else if ((ii + 1) % 3 == 0) {
        r += (ii + 1);
        System.out.print("Fizz");
      } else if ((ii + 1) % 5 == 0) {
        r += (ii + 1);
        System.out.print("Buzz");
      } else {
        System.out.print(ii + 1);
      }
    }
    System.out.println("Result: " + r);
  }
}
